spheres_in_the_sky/map.conf

```
@@ -0,0 +1,21 @@
+team.2.color = blue
+team.2.pos = (7,-49,-90)
+team.1.pos = (7,-49,89)
+team.1.color = red
+team.1 = red
+author = Daniel_Carter
+rotation = z
+h = 140
+team.2 = blue
+name = Spheres in The Sky
+initial_stuff = default:pick_steel,default:stick 2,default:torch 10,default:desert_cobble 70
+r = 115
+base_node = ctf_map:desert_cobble
+chests.1.n = 30
+chests.1.from = (-7.8,-45.5,-39.8)
+chests.1.to = (-22.2,-45.5,-55.2)
+chests.2.n = 30
+chests.2.from = (-4.8,-48.5,52.2)
+chests.2.to = (-19.2,-48.5,36.8)
```

```suggestion
chests.1.n = 30
chests.1.from = (-8,-45,-40)
chests.1.to = (-22,-46,-55)
chests.2.n = 30
chests.2.from = (-5,-49,52)
chests.2.to = (-19,-49,37)
```
Use only integer numbers

---

it wont work

---

why?

---

i tried and it didnt